crates/autopilot/src/shadow.rs

```diff
@@ -0,0 +1,270 @@
+use {
```

Comment:
A doc comment for this module would be nice (basically same info that's already part of the CLI `--help` text).

Reply:
c567f420c2b438976e12ace04484258a670c7577